app/app.go

```diff
@@ -414,11 +405,49 @@ func New(
 		),
 	)
 
+	baseBankKeeper := app.BankKeeper.(bankkeeper.BaseKeeper)
+	app.GravityKeeper = gravitykeeper.NewKeeper(
+		keys[gravitytypes.StoreKey],
+		app.GetSubspace(gravitytypes.ModuleName),
+		appCodec,
+		&baseBankKeeper,
+		app.StakingKeeper,
+		&app.SlashingKeeper,
+		&app.DistrKeeper,
+		&app.AccountKeeper,
+		&app.UIBCTransferKeeper.Keeper,
+		&app.bech32IbcKeeper,
+	)
+
+	// register the staking hooks
+	// NOTE: stakingKeeper above is passed by reference, so that it will contain these hooks
```

Comment:
I notice that some of the `&` on stakingKeeper above this comment have been removed in this PR. Is this intended?